testgres/node.py

```diff
@@ -629,9 +652,39 @@ def get_control_data(self):
 
         return out_dict
 
+    def slow_start(self, replica=False, dbname='template1', username=default_username()):
```

Comment:
why putting it to ProcessProxy? consider to put it to NodeApp maybe?

Reply:
Line 655, This function is already inside PostgresNode 
![image](https://user-images.githubusercontent.com/5807469/235120198-4f6e659a-f429-42ee-974f-eb460cd8c842.png)